### ISSUE TITLE
Shorten dredger build decay times or remve them completely.

### ISSUE THREAD

**Issue by mitm001** [SoftbearStudios/mk48#200]
Dredgers are now a problem as they are being used to section off huge swaths of the map. People are using 2 or 3 dredgers working in unison to partition the map down to nothing. 

The easiest remedy would be to speed up the decay time for dredged areas thereby forcing dredgers to constantly repair their areas. All their works should be completely decayed within 30 min to an hour. 30 min is what I would prefer myself.

Speeding up the decay decreases the overall area they can control.

**Comment by finnbear** [comment]
I agree, we'll shorten the times a little bit for the next update (I seem to remember already doing this, but it may be in the changes we haven't released yet).
